### ISSUE TITLE
Can't download models

### ISSUE THREAD

**Issue by mahmouds-cerebras** [microsoft/BioGPT#104]
Hello,
I get this error trying to download any of the models 

`PublicAccessNotPermittedPublic access is not permitted on this storage account. RequestId:730ef2c9-601e-0065-2bf1-974d08000000 Time:2023-06-05T21:02:26.5660150Z`

**Comment by Licko0909** [comment]
Me too

![image](https://github.com/microsoft/BioGPT/assets/27897166/6e5f7e50-b61a-44fc-8d46-56b243b1bf08)

**Comment by shashank140195** [comment]
I am facing the same issue.

**Comment by yx0516** [comment]
I am facing the same issue.

**Comment by lir0ni** [comment]
Me too! if anyone found a solution, pls share

**Comment by zhengjiangyu** [comment]
I'm feeling they left the pre-trained model in hugging face for downloading, it can be picked up by HF transformers but there were no checkpoints. So the pre-trained looks can not be finetuned due to lack of checkpoints missing...


fairseq-train \
    ../../data/BC5CDR/relis-bin --save-dir ${SAVE_DIR} \
    --user-dir ../../src \
    --finetune-from-model ../../checkpoints/Pre-trained-BioGPT/checkpoint.pt \   ### no such file even if adjusted to HF model repo
    --task language_modeling_prompt \
    --arch transformer_lm_prompt_biogpt \
    --share-decoder-input-output-embed --decoder-learned-pos \
    --optimizer adam --adam-betas '(0.9, 0.98)' \
    --weight-decay 0.01 --clip-norm 0.0 \
    --lr 1e-5 --lr-scheduler inverse_sqrt --warmup-updates 100 --warmup-init-lr 1e-07 \
    --tokens-per-sample 1024 --max-source-positions 640 --max-target-positions 1024 \
    --max-tokens 1024 --update-freq 32 \
    --skip-invalid-size-inputs-valid-test \
    --max-epoch 100 --keep-last-epochs 5 \
    --learned-prompt 9

**Comment by renqianluo** [comment]
Hi all, the link is fixed and you can try downloading again.

**Comment by lir0ni** [comment]
@renqianluo Hi, the error is back. Can you please fix it again?
